components/TextFields/styling/README.md

```diff
@@ -0,0 +1,77 @@
+#  Text field styles
+
+MDC's text fields are designed to be styled by controllers implementing the `MDCTextInputController` protocol. The following controllers are included in the text field component and each one is highly customizeable to express your app's branding and styling.
+
+Some of the classes are marked as "Preferred". These are known, thru extensive UX research, to have big performance gains in usability.
+
+Another two of the classes are "Legacy". They are included to support older designs and avoid breaking changes that come from the size of newer styles.
```

Comment:
Grammar nit: drop "Another". Or possibly "Two of classes include 'Legacy' in their names."

Reply:
Done.